apps/coscrad-frontend/src/components/transcripts/transcript-presenter.tsx

```diff
@@ -18,7 +18,7 @@ export const TranscriptPresenter = ({ transcript }): JSX.Element => {
                 {isNullOrUndefined(transcript)
                     ? null
                     : transcript.participants.map((item: ITranscriptParticipant) => (
-                          <Box>
+                          <Box key={item.name.replace(' ', '-')}>
```

Comment:
I'm assuming the `ITranscriptParticipant` also has `initials`. Given that these server as a "locally unique identifier" for the transcript participant, use the initials as a key instead.

Reply:
Ok great, I was wondering about that.  I'll change it up.